frontend/src/metabase/querying/notebook/components/NotebookDataPicker/NotebookDataPicker.tsx

```diff
@@ -44,10 +45,15 @@ export function NotebookDataPicker({
   table,
   databaseId,
   placeholder = title,
-  hasMetrics,
+  hasMetrics = false,
   isDisabled,
   onChange,
 }: NotebookDataPickerProps) {
+  const { modelsFilterList } = useNotebookContext();
```

Comment:
Now, when `hasMetrics` is `false`, we exclude `metric`.

The default behavior is that we have `metric` in our `modelsFilterList` array, so if `hasMetrics` is true, `metric` will be there